packages/rancher-monitoring/package.yaml

```diff
@@ -2,4 +2,5 @@ url: https://kubernetes-charts.storage.googleapis.com/prometheus-operator-8.16.1
 packageVersion: 00
 generateCRDChart:
   enabled: true
-  providesGVR: prometheuses.monitoring.coreos.com/v1
\ No newline at end of file
+  providesGVR: prometheuses.monitoring.coreos.com/v1
+  assumeOwnershipOfCRDs: true
```

Comment:
Since the bash scripts are getting so complicated already, I wonder if this sort of thing should just exist in overlays in the monitoring chart, as its a fairly unique situation. Only a few cases could possible exist.

Reply:
The issue with using `overlay/` is that these files aren't being added to the main chart but rather to a separate CRD chart. However, if the resolution for https://github.com/rancher/charts/pull/595#discussion_r480376364 is to avoid creating a separate CRD chart and instead just have the Job exist in the main chart, I think this can be done as a one-off for monitoring.

Reply:
Based on https://github.com/rancher/charts/pull/595#issuecomment-685861749, the overlay/ solution is no longer possible.